app/managers.py

```diff
@@ -0,0 +1,41 @@
+import sqlite3
+
+from app.models import Actor
+
+
+class ActorManager:
+    def __init__(self):
+        self._connection = sqlite3.connect("cinema.db3")
+        self.table_name = "actors"
+
+    def create(self, first_name: str, last_name: str):
+        self._connection.execute(
+            f"INSERT INTO {self.table_name} (first_name, last_name)"
+            f"VALUES (?, ?)", (first_name, last_name)
+        )
+        self._connection.commit()
+
+    def all(self):
+        actor_cursor = self._connection.execute(
+            f"SELECT id, first_name, last_name FROM {self.table_name}"
+        )
+
+        return [Actor(*row) for row in actor_cursor]
+
+    def update(self, id_to_update: int,
+               new_first_name_: str, new_last_name: str):
```

Comment:
What for do you add `_` at the end of a variable name?

Reply:
Samurai have no goal, only way